fresnel/geometry.py

```diff
@@ -183,6 +183,171 @@ def radius(self):
     def color(self):
         return util.array(self._geometry.getColorBuffer(), geom=self)
 
+
+class Box(Cylinder):
+    R""" Box geometry.
+
+    Generate a triclinic box shape.
+
+    Args:
+        scene (:py:class:`fresnel.Scene`): Add the geometry to this scene
+        box (`numpy.ndarray` or `array_like`): (``1x1, 1x3, or 1x6`` : ``float32``) Assumes 1x1 is cubic, 1x3 is orthorhombic, and 1x6 is triclinic.
+        radius (``float32``): Radius of box edges.
+        color (`numpy.ndarray` or `array_like`): (``1x3`` : ``float32``) Color of box edges.
+
+    .. seealso::
+
+        :doc:`examples/01-Primitives/05-Box-geometry`
+            Tutorial: defining and setting box geometry properties
+        :doc:`examples/02-Advanced-topics/05-GSD-visualization`
+            Tutorial: Visualizing GSD files
+
+
+    Note:
+        The constructor arguments ``radius``, and ``color`` are optional. If you do not provide them,
+        they are initialized to 0.5 and black, respectively.
+
+    .. hint::
+        Avoid costly memory allocations and type conversions by specifying primitive properties in the appropriate
+        numpy array type.
+
+    Attributes:
+        points (:py:class:`fresnel.util.array`): Read the points at the corners of the box.
+        radius (:py:class:`fresnel.util.array`): Read or modify the radii of the box edges.
+        color (:py:class:`fresnel.util.array`): Read or modify the colors of the box.
```

Comment:
These are inherited from `Cylinder`. Their documentation should either match that in `Cylinder` or should be removed to indicate we don't intend the user to access these. Also, document the `box` attribute.